src/managing-workflow/platform-logging.md

```diff
@@ -13,8 +13,8 @@ Logger acts like a syslog server and receives all log messages that are occurrin
 Installation of the logging components is separate from the installation of the Deis platform because of the requirement of Daemon Sets. You will need to follow these steps to get the logging components running on your kubernetes cluster.
 
 ```
-$ helm fetch deis/deis-logger
-$ helm install deis-logger
+$ helm fetch deis/deis-logger-beta1-test
```

Comment:
I think the `-test` prefix will go away once we officially cut beta (Maybe I'm confused?)